packages/ui/src/Integration/IntegrationDetailHistoryListView.tsx

```diff
@@ -14,10 +16,14 @@ export class IntegrationDetailHistoryListView extends React.Component<
 > {
   public render() {
     return (
-      <Grid fluid={true} key={1}>
+      <Grid
+        fluid={true}
+        key={1}
+        className="integration-detail-history-list-view"
+      >
         {this.props.integrationIsDraft ? (
           <Grid.Row className="show-grid">
-            <Grid.Col xs={2} md={2}>
+            <Grid.Col xs={2} md={2} className="list-description">
```

Comment:
I think we should have a class name that ties this to this component, as by BEM convention